addons/web/static/src/legacy/xml/kanban.xml

```diff
@@ -20,12 +20,12 @@
                     <div class="dropdown-menu" role="menu">
                         <a role="menuitem" class="dropdown-item o_kanban_toggle_fold" href="#">Fold</a>
                         <t t-if="widget.grouped_by_m2o">
-                            <a t-if="widget.editable and widget.id" role="menuitem" class="dropdown-item o_column_edit" href="#">Edit Stage</a>
+                            <a t-if="widget.editable and widget.id" role="menuitem" class="dropdown-item o_column_edit" href="#">Edit</a>
                             <a t-if="widget.deletable and widget.id" role="menuitem" class="dropdown-item o_column_delete" href="#">Delete</a>
                         </t>
                         <t t-if="widget.has_active_field and widget.archivable">
-                            <a role="menuitem" href="#" class="dropdown-item o_column_archive_records">Archive All</a>
-                            <a role="menuitem" href="#" class="dropdown-item o_column_unarchive_records">Unarchive All</a>
+                            <a role="menuitem" href="#" class="dropdown-item o_column_archive_records">Archive All Tasks</a>
+                            <a role="menuitem" href="#" class="dropdown-item o_column_unarchive_records">Unarchive All Tasks</a>
```

Comment:
Are you sure it's what you want to do ? You change the xml of the kanban view globally here, meaning that your change will impact all the kaban views in any modules.
If you want to only do it in your module, you'll need to overwrite this template instead :)